index.js

```diff
@@ -27,29 +38,84 @@ const urls = fs
   .split(/\r?\n/)
   .map((l) => `${l}`);
 
-const addonUUID = 'd56a5b99-51b6-4e83-ab23-796216679614';
+function whichBrowserIsSelected(isChromeSelected) {
+  return isChromeSelected
+    ? 'agbomdmjjfglgplpnkahbcmblehajkag'
+    : 'd56a5b99-51b6-4e83-ab23-796216679614';
+}
+let addonUUID = whichBrowserIsSelected(isChromeSelected);
```

Comment:
no need for a function 
```suggestion
const addonUUID = isChromeSelected
    ? 'agbomdmjjfglgplpnkahbcmblehajkag'
    : 'd56a5b99-51b6-4e83-ab23-796216679614';
```

Reply:
Deleted. addonUUID is set different way for Chrome.